Source/Urho3D/UI/UI.h

```diff
@@ -163,7 +165,10 @@ class URHO3D_API UI : public Object
 
     /// Return UI element double click interval in seconds.
     float GetDoubleClickInterval() const { return doubleClickInterval_; }
-
+    
+    /// Get max screen distance for double clicks to register.  default is FLT_MAX.

```

Comment:
Comment not properly case. The constant value seems to be different than the implementation.